### DISCUSSION TITLE
Smiley Player Trader Release 1.7.11

### DISCUSSION THREAD

**Discussion by Mrcomputer1** [Mrcomputer1/SmileyPlayerTrader#63] [Releases]
# Change Log
* Fix: Adds way to see if the admin unlimited supply option is enabled or disabled for an item.

**Note: We recommend backing up your SPT database and configuration file, just in case. While it is unlikely something will go wrong, it is always a good idea to have backup.**

<hr /><em>This discussion was created from the release <a href='https://github.com/Mrcomputer1/SmileyPlayerTrader/releases/tag/v1.7.11'>Smiley Player Trader Release 1.7.11</a>.</em>

**Comment by TheRealFosya** [comment]
[ru_ru.json](https://github.com/user-attachments/files/16227920/ru_ru.json)
Updated translation on new functions.

**Reply by Mrcomputer1** [reply]
Thank you. I will release a version with the updated translation tomorrow.

**Reply by TheRealFosya** [reply]
For you to know, I will support translation for your plugin as much as i can, bc i love it, and i love that fast update request response to users that you providing. Thank you.

**Reply by TheRealFosya** [reply]
I updated a file again, i fixed not translated "page" word on line 51.

**Reply by Mrcomputer1** [reply]
> For you to know, I will support translation for your plugin as much as i can, bc i love it, and i love that fast update request response to users that you providing. Thank you.

Thank you.

> I updated a file again, i fixed not translated "page" word on line 51.

Ok, thanks, I will release a version with the updated translation file soon.
